ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -191,13 +191,14 @@ vacuum_cost_delay = 20ms        # 0-100 milliseconds
                 # number of seconds; 0 disables
 
 {% if hot_standby_server|default(None) %}
-wal_level = hot_standby
 archive_mode = on
 archive_command = 'rsync -a %p postgres@{{ hot_standby_server }}:{{ archive_dir }}/%f'   # archive directory lives on first standby machine specified in the inventory file
 
-max_wal_senders = 3
 {% endif %}
 
+wal_level = hot_standby
+max_wal_senders = 3
+
```

Comment:
This allows us to use the same backup scripts on clusters that have a standby and ones that don't, and according to postgres docs, raising the `wal_level` should not have an impact on performance.